spec/services/govuk_emails/monitor_spec.rb

```diff
@@ -42,26 +42,65 @@
 
         it "schedules an undeliverable alert email" do
           expect { subject }.to change(ScheduledMailing, :count).by(1)
-          undeliverable_mail = ScheduledMailing.order(:created_at).last
-
-          expect(undeliverable_mail.legal_aid_application_id).to eq scheduled_mailing.legal_aid_application_id
-          expect(undeliverable_mail.mailer_klass).to eq "UndeliverableEmailAlertMailer"
-          expect(undeliverable_mail.mailer_method).to eq "notify_apply_team"
-          expect(undeliverable_mail.arguments).to eq [scheduled_mailing.id]
-          expect(undeliverable_mail.scheduled_at).to have_been_in_the_past
-          expect(undeliverable_mail.addressee).to eq Rails.configuration.x.support_email_address
+          undeliverable_alert_mail = ScheduledMailing.order(:created_at).last
+
+          expect(undeliverable_alert_mail.legal_aid_application_id).to eq scheduled_mailing.legal_aid_application_id
+          expect(undeliverable_alert_mail.mailer_klass).to eq "UndeliverableEmailAlertMailer"
+          expect(undeliverable_alert_mail.mailer_method).to eq "notify_apply_team"
+          expect(undeliverable_alert_mail.arguments).to eq [scheduled_mailing.id]
+          expect(undeliverable_alert_mail.scheduled_at).to have_been_in_the_past
+          expect(undeliverable_alert_mail.addressee).to eq Rails.configuration.x.support_email_address
+        end
+      end
+
+      context "when the failed email is a citizen start email" do
+        let(:applicant) { create(:applicant, email: "johndoe@example.net") }
+        let(:provider) { create(:provider) }
+        let(:legal_aid_application) { create(:legal_aid_application, applicant:, provider:) }
+        let(:scheduled_mailing) do
+          create(:scheduled_mailing,
+                 :citizen_start_email,
+                 :processing,
+                 addressee: applicant.email,
+                 legal_aid_application_id: legal_aid_application.id)
+        end
+        let(:mailer_args) do
+          [
+            provider.email,
+            legal_aid_application.application_ref,
+            applicant.full_name,
+            applicant.email,
+          ]
+        end
+
+        before { allow(HostEnv).to receive(:production?).and_return(true) }
+
+        it "schedules an email to notify the provider of the failure" do
+          expect { subject }.to change(ScheduledMailing, :count).by(2)
+        end
+
+        it "uses the undeliverable email alert mailer and the notify_provider method with the right args" do
```

Comment:
aggregate_failures could be used here to

```suggestion
        it "uses the undeliverable email alert mailer and the notify_provider method with the right args", :aggregate_failures do
```